static/app/components/nav/sidebar.tsx

```diff
@@ -1,9 +1,11 @@
-import {Fragment} from 'react';
+import {Fragment, type PropsWithChildren} from 'react';
```

Comment:
A word of caution, but PropsWithChildren doesn't mean that the children are actually required, which I think is what you actually want here?
```
type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };
```

For example in this case, you want the children to always be present right?
```
  return (
    <StyledLink
      {...linkProps}
      className={isActive || isSubmenuActive ? 'active' : undefined}
      aria-current={isActive ? 'page' : undefined}
    >
      <InteractionStateLayer hasSelectedBackground={isActive || isSubmenuActive} />
      {children}
    </StyledLink>
```

Reply:
That's annoying, thanks for the callout!